tests/marine/test_marine.py

```diff
@@ -961,9 +961,9 @@ def test_validate_fields_success(marine_instance: Marine):
 
 
 def test_validate_fields_failure(marine_instance: Marine):
-    assert not marine_instance.validate_fields(
-        ["ip.src", "eth.dst", "this.field.is.bad"]
-    )
+    result = marine_instance.validate_fields(["ip.src", "eth.dst", "this.field.is.bad"])
```

Comment:
I think this test can be improved to better demonstrate the error behavior. For example, the array should contain a mix of good and bad fields `[good1, bad1, good2, bad2, ...]`. Does the error return all of the bad fields or does it fail after the first one? Is the order of the bad fields kept in the error message?